unyt/_array_functions.py

```diff
@@ -926,11 +931,9 @@ def einsum(subscripts, *operands, out=None, **kwargs):
     ret_units = _validate_units_consistency(operands)
 
     if out is not None:
-        out_view = out.view(np.ndarray)
+        res = np.einsum._implementation(subscripts, *operands, out=out.view(np.ndarray))
```

Comment:
this patch made me realise that this wrapper is swallowing kwargs other than `out` silently